tests/kubevirt_helper.go

```diff
@@ -762,3 +763,19 @@ func DeployVMTemplatesAndValidate() error {
 	_, err = task.DoRetryWithTimeout(waitForCompletedAnnotations, importerPodCompletionTimeout, importerPodRetryInterval)
 	return err
 }
+
+func AddToVMNodeMap(vmName, nodeName string, vmNodeMap *map[string]string) {
```

Comment:
nit: maybe get rid of this function. It does not seem to do much.

Reply:
Fixed.